runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotPlugin.java

```diff
@@ -751,4 +776,10 @@ int gettheatreOfBloodNumber()
 	{
 		return theatreOfBloodNumber;
 	}
+
+	@VisibleForTesting
+	String getScreenShotSubDir()
```

Comment:
I guess this can be removed since it's no longer tested?

Reply:
Ya that is no longer need. I will remove it.